students/prgrover/lesson07/html_render.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+class Element:
+    """
+    Base class for rendering an HTML element
+    """
+
+    tag = 'html'
+    indent = '    ' #4 space characters
+
+    def __init__(self, content=None, **kwargs):
+        self.attributes = kwargs
+        self.content = []
+        if content:
+            self.append(content)
+
+
+    def append(self, content):
```

Comment:
Pep8 specifies 2 blank lines between classes and module level functions and one blank line between methods